app/templates/events/event_list.html

```diff
@@ -96,12 +96,16 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
               {% set link_class = 'link-secondary' if event.isPast else 'link-primary' %}
                 <td><a href="/event/{{event.id}}/edit" class="{{link_class}}">{{event.name}}</a></td>
             {% else %}
-              {% set link_class = 'link-secondary' if event.isPast else 'link-primary' %}
-              <td><a href="/event/{{event.id}}/view" class="{{link_class}}">{{event.name}}</a></td>
+              <td><a href="/event/{{event.id}}/view" class="link-primary">{{event.name}}</a></td>
             {% endif %}
-            <td nowrap>{{event.startDate.strftime('%m/%d/%Y')}}</td>
-            <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
-            <td class="text-dark">{{event.location}}</td>
+            {% set ProgramManager = 'programmanager' if isProgramManager  else 'CELTS' %}
```

Comment:
This is putting the text "programmanager" as the contact instead of the actual contact info. Also, variables shouldn't start with a capital letter.

Reply:
Fixed